internal/cli/update/update.go

```diff
@@ -0,0 +1,99 @@
+package update
+
+import (
+	"errors"
+	"fmt"
+
+	"connectrpc.com/connect"
+	pkgerrors "github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/utils/pointer"
+	sigyaml "sigs.k8s.io/yaml"
+
+	"github.com/akuity/kargo/internal/cli/client"
+	"github.com/akuity/kargo/internal/cli/option"
+	kargosvcapi "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+)
+
+type Flags struct {
+	Filenames []string
+}
+
+func NewCommand(opt *option.Option) *cobra.Command {
+	var flag Flags
+	cmd := &cobra.Command{
+		Use:   "update [--project=project] -f (FILENAME)",
```

Comment:
Wondering if the sub-command should perhaps be `apply` for improved consistency with kubectl -- since users are presumably already very familiar with that.

Reply:
@devholic is apply different from update?